app/src/components/CreateEvent/CreateEventContainer.tsx

```diff
@@ -2,66 +2,59 @@ import { useState } from 'react';
 import React from 'react';
 import {
   IEditEvent,
-  parseEvent,
-  IEvent,
-  initialEditEvent,
+  initialEvent,
+  toEditEvent,
+  parseEditEvent,
 } from 'src/types/event';
 import { postEvent } from 'src/api/arrangementSvc';
-import { isOk, Result } from 'src/types/validation';
 import { useHistory } from 'react-router';
 import { viewEventRoute, eventsRoute, editEventRoute } from 'src/routing';
 import { useAuthentication } from 'src/auth';
-import style from './CreateEventContainer.module.scss';
 import { useSavedEditableEvents } from 'src/hooks/eventHooks';
 import { useNotification } from 'src/components/NotificationHandler/NotificationHandler';
 import { Page } from 'src/components/Page/Page';
 import { PreviewEventContainer } from 'src/components/PreviewEvent/PreviewEventContainer';
 import { Button } from 'src/components/Common/Button/Button';
 import { EditEvent } from 'src/components/EditEvent/EditEvent/EditEvent';
 import { BlockLink } from 'src/components/Common/BlockLink/BlockLink';
-import { parseQuestion } from 'src/types';
+import style from './CreateEventContainer.module.scss';
+import { isValid } from 'src/types/validation';
 
 export const CreateEventContainer = () => {
   useAuthentication();
-
-  const [event, setEvent] = useState<Result<IEditEvent, IEvent>>(
-    parseEvent(initialEditEvent())
-  );
   const [previewState, setPreviewState] = useState(false);
-  const isDisabled = !isOk(event);
+
+  const [event, setEvent] = useState<IEditEvent>(toEditEvent(initialEvent()));
+  const parsedEvent = parseEditEvent(event);
+
   const history = useHistory();
   const { catchAndNotify } = useNotification();
   const { saveEditableEvents } = useSavedEditableEvents();
 
   const addEvent = catchAndNotify(async () => {
-    if (isOk(event)) {
-      if (event.validValue.participantQuestion === '') {
-        updateEvent({
-          ...event.editValue,
-          participantQuestion: parseQuestion(
-            'Allergier, preferanser eller noe annet på hjertet?'
-          ),
-        });
```

Comment:
Samme her, må settes default en eller annen plass, og det var ikke noe bra plass for det.